statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -646,20 +828,35 @@ def bic(self):
     @cache_readonly
     def cov_params_default(self):
```

Comment:
Based on the previous comments: At least you should try to work more "with" instead of "against" the inherited structure. I haven't checked yet how this plays together with what's in LikelihoodModelResults.
(My impression is now that it's dangerous to work halfway consistently with the inherited structure because it might trigger some of the `hasattr` paths.)